video/out/opengl/context_wayland.c

```diff
@@ -56,7 +81,39 @@ static void resize(struct ra_ctx *ctx)
 static void wayland_egl_swap_buffers(struct ra_ctx *ctx)
 {
     struct priv *p = ctx->priv;
+    struct vo_wayland_state *wl = ctx->vo->wl;
+
+    struct pollfd fds[2] = {
+        {.fd = wl->display_fd,     .events = POLLIN },
+        {.fd = wl->wakeup_pipe[2], .events = POLLIN },
```

Comment:
I think including the "main" wakeup pipe is wrong. It shouldn't give up on drawing the frame just because the player frontend (or whatever) wants something.

Reply:
Actually I think this is never set, maybe? Wouldn't that mean poll always returns with an error, making this a busy loop?

Reply:
You're right, I only need the display_fd.